.gitmodules

```diff
@@ -0,0 +1,4 @@
+[submodule "glTF-Blender-IO"]
+	path = glTF-Blender-IO
+	url = https://github.com/KhronosGroup/glTF-Blender-IO.git
```

Comment:
Submodule should point on sketchfab, not KhronosGroup.

The idea is to "reset --hard" our master branch on khronos one, once we archived our repository